liquibase.properties

```diff
@@ -0,0 +1,5 @@
+changelogFile=./liquibase/happraisal.yml
+driver=org.postgresql.Driver
+url=jdbc:postgresql://localhost:5432/happraisal
```

Comment:
Could you please change the name of the database according to the configuration?

Reply:
done